--- ../src_base/minecraft/net/minecraft/client/entity/AbstractClientPlayer.java
+++ ../src_work/minecraft/net/minecraft/client/entity/AbstractClientPlayer.java
@@ -18,9 +18,9 @@
 {
     public static final ResourceLocation locationStevePng = new ResourceLocation("textures/entity/steve.png");
     private ThreadDownloadImageData downloadImageSkin;
-    private ThreadDownloadImageData downloadImageCape;
+    public ThreadDownloadImageData downloadImageCape;
     private ResourceLocation locationSkin;
-    private ResourceLocation locationCape;
+    public ResourceLocation locationCape;
 
     public AbstractClientPlayer(World par1World, String par2Str)
     {
