--- AbstractClientPlayer.java	2013-07-21 09:15:36.000000000 -0500
+++ AbstractClientPlayer.java	2013-07-20 15:29:36.000000000 -0500
@@ -18,9 +18,9 @@
 {
     public static final ResourceLocation field_110314_b = new ResourceLocation("textures/entity/steve.png");
     private ThreadDownloadImageData field_110316_a;
-    private ThreadDownloadImageData field_110315_c;
+    public ThreadDownloadImageData field_110315_c;
     private ResourceLocation field_110312_d;
-    private ResourceLocation field_110313_e;
+    public ResourceLocation field_110313_e;
 
     public AbstractClientPlayer(World par1World, String par2Str)
     {
